suite-common/wallet-utils/src/getMineInputsFromTransaction.ts

```diff
@@ -0,0 +1,35 @@
+import { AccountTransaction } from '@trezor/connect';
+import { Account } from '@suite-common/wallet-types';
+
+export type GetMineInputsFromTransactionParams = {
+    account: Pick<Account, 'addresses'>;
+    tx: Pick<AccountTransaction, 'details'>;
+};
+
+export const getMineInputsFromTransaction = ({
```

Comment:
```suggestion
export const getMyInputsFromTransaction = ({
```

Also in the file name.